docs/source/models.rst

```diff
@@ -912,6 +912,116 @@ interpreted as generic Python functions for inference via :py:func:`mlflow.pyfun
 only be scored with DataFrame input. You can also use the :py:func:`mlflow.fastai.load_model()` method to
 load MLflow Models with the ``fastai`` model flavor in native fastai format.
 
+The interface for utilizing a ``fastai`` model loaded as a pyfunc type for generating predictions uses a 
+Pandas DataFrame argument. 
+
+This example runs the `fastai tabular tutorial <https://docs.fast.ai/tutorial.tabular.html>`_, 
+logs the experiments, saves the model in ``fastai`` format, and loads the model to get predictions
+using a ``fastai`` data loader:
+
+.. code-block:: py
+
+    from fastai.tabular.all import *
```

Comment:
Do we need to do a wildcard import here? Can you adjust to using named entities here?

Reply:
The fast.ai examples mostly include a wildcard import in their examples (see https://docs.fast.ai/tutorial.tabular.html). If you want to avoid that, I will change to named entities.

Reply:
+1 to avoiding that.

Reply:
I pushed a new commit that addresses this.